include/eve/detail/shuffle_v2/simd/x86/shuffle_l2.hpp

```diff
@@ -279,39 +281,56 @@ requires (P::out_reg_size == P::reg_size)
 
 template<typename P, arithmetic_scalar_value T, typename N, std::ptrdiff_t G>
 EVE_FORCEINLINE auto
-shuffle_l2_(EVE_SUPPORTS(sse2_), P, fixed<G>, wide<T, N> x, wide<T, N> y)
-requires std::same_as<abi_t<T, N>, x86_128_> && (P::out_reg_size == P::reg_size)
+shuffle_l2_x86_blend(P, fixed<G>, wide<T, N> x, wide<T, N> y)
 {
   // Immediate blends are very good, even if are covered by other ops
   // https://stackoverflow.com/questions/76552874/how-should-i-chose-between-mm-move-sd-mm-shuffle-pd-mm-blend-pd
   //
-  // FIX-1617 - enable `_mm_blend_epi16`
```

Comment:
is 1617 still relevant then ?

Reply:
Yes - I just moved this comment to a different place